docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -401,6 +398,24 @@ mvn -Dmaven.javadoc.skip=true --projects tinkergraph-gremlin test
 ** Generate web site locally: `bin/generate-home.sh`
 ** Publish web site: `bin/publish-home.sh <username>`
 
+[[building-on-windows]]
+== Building On Windows
+
+The following are the steps a Windows user must take in order to Build Tinkerpop on Windows:
+
+. Install winutils for Hadoop
+* Download the link:https://github.com/cdarlint/winutils/blob/master/hadoop-2.7.7/bin/winutils.exe[winutils.exe] file for Hadoop version `2.7.7` from cdarlint/winutils
+* Place `winutils.exe` file wherever you would like to in the following folder structure:
+** `hadoop-2.7.7/bin/winutils.exe`
+* Set `HADOOP_HOME` to point to `hadoop-2.7.7` folder
```

Comment:
Does it make sense to have this part be more generic rather than to refer to Hadoop version 2.7.7? Maybe just tell them to grab the required hadoop version for their branch?

Reply:
the Hadoop version is fixed in `3.5` to `2.7.7`

Reply:
may be better to push different text for 3.5 and 3.6?